src/Compilers/CSharp/Portable/Binder/Binder_WithExpression.cs

```diff
@@ -95,85 +94,51 @@ private BoundExpression BindWithExpression(WithExpressionSyntax syntax, Diagnost
             }
 
             var cloneReturnType = cloneMethod?.ReturnType;
-
-            var args = ArrayBuilder<(Symbol?, BoundExpression)>.GetInstance();
+            // PROTOTYPE: Handle dynamic
+            var implicitReceiver = new BoundObjectOrCollectionValuePlaceholder(
+                syntax.Receiver,
+                // formatting
+            #pragma warning disable IDE0055
```

Comment:
>#pragma warning disable IDE0055 [](start = 12, length = 31)

nit: What's wrong with the formatting? Should we file an issue to refine that analyzer?